main.tf

```diff
@@ -229,9 +247,21 @@ resource "aws_cloudfront_distribution" "default" {
       domain_name = origin.value.domain_name
       origin_id   = origin.value.origin_id
       origin_path = lookup(origin.value, "origin_path", "")
+
+      dynamic "custom_header" {
```

Comment:
How come the same custom_header isn't applied to the var.s3_origins origin block below ?